### PR TITLE
Be less chatty when we're uploading to GCS

### PR DESCRIPTION
**Pull request by zmerlynn**

### CODE REVIEW

**Comment by zmerlynn** [comment]
cc @ixdy

File: build/common.sh [hunk 1]
```diff
@@ -814,11 +814,11 @@ function kube::release::gcs::copy_release_artifacts() {
   fi
 
   # Now upload everything in release directory
-  gsutil -m "${gcs_options[@]+${gcs_options[@]}}" cp -r "${RELEASE_DIR}"/* "${gcs_destination}"
+  gsutil -q -m "${gcs_options[@]+${gcs_options[@]}}" cp -r "${RELEASE_DIR}"/* "${gcs_destination}"
 
   # Having the "template" scripts from the GCE cluster deploy hosted with the
   # release is useful for GKE.  Copy everything from that directory up also.
-  gsutil -m "${gcs_options[@]+${gcs_options[@]}}" cp \
```
Comment (ixdy) [inline build/common.sh]: did you mean to remove -m here?

**Comment by ixdy** [comment]
I also wonder if we only want to pass -q when running in Jenkins.

File: build/common.sh (hunk 1 above)
Comment (zmerlynn) [inline build/common.sh]: I did. The multithreading doesn't help for a single file. (I guess it doesn't hurt to leave it, though.)

**Comment by zmerlynn** [comment]
It's pretty spammy regardless. I guess you could say "interactive", but honestly ...

**Comment by ixdy** [comment]
LGTM

**Pull request merged**